bokehjs/src/lib/models/widgets/file_input.ts

```diff
@@ -16,43 +19,68 @@ export class FileInputView extends WidgetView {
     if (this.dialogEl == null) {
       this.dialogEl = document.createElement('input')
       this.dialogEl.type = "file"
-      this.dialogEl.multiple = false
-      this.dialogEl.onchange = (e) => this.load_file(e)
+      this.dialogEl.multiple = false  
+      this.dialogEl.onchange = async (e) => await this.load_files(e)
       this.el.appendChild(this.dialogEl)
     }
     if (this.model.accept != null && this.model.accept != '')
       this.dialogEl.accept = this.model.accept
+    if (this.model.multiple == 'multiple' || this.model.multiple == 'True'){
+      this.dialogEl.multiple = true
+    }
```

Comment:
It would have been much better, if `multiple` property was defined as `bool`, then you could just assign it to the DOM element's attribute on the original line 19. In general make your APIs as strict or as narrow as possible.

Reply:
Agree on the Bokeh end `multiple` should just accept True/False

Reply:
No problem, bool was invented for such cases. I used it like that to have consistency with other codes I found like HTML5 or some JS gists.
I'll change it to bool.

Reply:
Done/Implemented in PR #9732